base/rhsso/deploy-rhsso-job.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: deploy-rhsso-operator
+spec:
+  template:
+    spec:
+      serviceAccountName: deploy-rhsso-operator
+      containers:
+        - name: deploy-rhsso-operator
+          image: docker.io/bitnami/kubectl:latest
```

Comment:
if possible please use image from quay.io

Reply:
there is no official image for quay, only option is to rebuild